### ISSUE TITLE
Tensorflow 2.0 add regularization losses

### ISSUE THREAD

**Issue by moshimo2019** [tensorflow/tensorflow#30866]
<em>In Tensorflow 1.x, I can add regularization losses by using code like this:
`regularization_loss = tf.add_n(tf.losses.get_regularization_losses(), 'regu')`
`total_loss = loss + regularization_loss`
But in tensorflow 2.0.0beta1 api, the 'losses.get_regularization_losses()' was canceled.So how can I add that loss In this case?

**System information**
- TensorFlow version :2.0.0beta1

**Comment by jvishnuvardhan** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged//tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

**Comment by jvishnuvardhan** [comment]
Closing this as it is support issue. Thanks!

**Comment by srihari-humbarwadi** [comment]
TF2.0 style for achieving the same. @moshimo2019 
```
regularization_loss = tf.math.add_n(model.losses)
```

**Comment by Jessespace** [comment]
> regularization_loss = tf.math.add_n(model.losses)

I used this style, but model.losses return [],Why?

**Comment by Yablon** [comment]
@Jessespace Do you set the kernel_regularizer of the layer ? See the doc [tensorflow.keras.layers.Dense](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Dense)
